Makefile

```diff
@@ -38,13 +38,19 @@ docs:
 	make -C docs
 
 .PHONY: test
-test:
```

Comment:
Are you sure you want pip install in the Makefile?

Like we only need to install that once

Reply:
test/ci.sh in the Makefile seems a little questionable too...

I'd expect one to be able to run "make test" locally without it attempting to install loads of things...

Reply:
But just to test this out, need to put the pip install's before codespell

Reply:
If it was me I'd do dependency installs in:

test/ci.sh

this could call:

make test

and:

make install (and this would only install ramalama python scripts, not dependancies)

Reply:
I think we can make the requirement for codespel and autopep8 smarter in the Makefile.  I would like to get to the point where the CI/CD system just runs make test. Then running it locally or in the CI/CD system make sense.

What I don't understand now is why the install.py is blowing up.